### ISSUE TITLE
Typo in tutorial? Wrong version to install with conda

### ISSUE THREAD

**Issue by melparker101** [B-UMMI/chewBBACA#201]
Hello,

In the [tutorial](https://chewbbaca.readthedocs.io/en/latest/user/getting_started/installation.html), the installation instructions tell us to install version 3.3.7 with conda. Is this a typo? It looks like the latest version is 3.3.6 (added to bioconda just 2 weeks ago).

I think this should be changed to 
```
conda create -c bioconda -c conda-forge -n chewie "chewbbaca=3.3.6"
```

Many thanks
Melody

**Comment by rfm-targa** [comment]
Greetings,

We are releasing v3.3.7 today and have already changed the version in the scripts and documentation.
Sorry about that. It will be available soon.

Best regards,

Rafael

**Comment by melparker101** [comment]
Hello @rfm-targa,

Looking forward to it!

Many thanks
Melody

**Comment by rfm-targa** [comment]
Greetings,

Released v3.3.7 and v3.3.8 right after. Closing this issue now. Let us know if there is anything else.

Best regards,

Rafael
